packages/ui/src/components/DatePicker/PreviousDateRangePicker.tsx

```diff
@@ -60,6 +64,64 @@ function toDateTimeString(date: Date, showYear: boolean) {
 	return date.toLocaleDateString('en-us', options)
 }
 
+/**
+ * Takes in a time string in the format HH:mm aa or HH:mmaa and returns	an object with the hour, minute, time of day, and 24 hour time
+ * @param timeString
+ * @returns {	hour: number, minute: number, timeOfDay: string, hour24: number}
+ */
+export const getTimeInfo = (timeString: string) => {
+	const date = moment(timeString, 'hh:mm a')
+	return {
+		hour: date.format('HH'),
+		minute: date.minute(),
+		timeOfDay: date.hour() >= 12,
+		hour24: date.hour(),
+	}
+}
```

Comment:
![image](https://user-images.githubusercontent.com/1351531/234446074-81cf2bd4-fd85-4ab2-8956-b694ff246e93.png)

just noticed this actually isn't quite working for AM/PM times (it works right for 24-hour times like 12:00 or 14:00). might be that the moment format needs to be different, could you please double check?

Reply:
fyi just tried with 89ed966d297ea4ea607776271f58d871b87460a9 and this still isn't quite working.
we should also support inputs like `6pm` or `9 PM`